speech-api/SpeechSynthesisEvent-properties.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+<title>Properties of SpeechSynthesisEvent</title>
+<link rel="help" href="https://wicg.github.io/speech-api/#utterance-events">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<body>
+<script>
+// Written for https://github.com/WICG/speech-api/issues/103
+promise_test(async (t) => {
+  const utterance = new SpeechSynthesisUtterance('test');
+  const eventWatcher = new EventWatcher(t, utterance, ['start', 'end', 'error']);
+  await test_driver.bless('speechSynthesis.speak',
+      () => speechSynthesis.speak(utterance));
```

Comment:
Just so it's clear that speak doesn't actually return anything. 
```suggestion
  await test_driver.bless('speechSynthesis.speak');
  speechSynthesis.speak(utterance);
```

Reply:
The 2nd argument here is intended to be used like this, the callback for the thing that needs a user gesture. But I can wrap in `{}` so that it doesn't matter what `speechSynthesis.speak(utterance)` returns.

Reply:
Understood. I generally don't bother using the bless' callback, as the click() resolves the bless' promise (enabling the transient activation). Just makes the code a bit easier on the eye. 

But again, just a nit. All looks good 👍